lib/hostip.c

```diff
@@ -68,6 +68,10 @@
 #include "curl_memory.h"
 #include "memdebug.h"
 
+#ifdef USE_RESOLVE_ON_IPS
+#include <SystemConfiguration/SystemConfiguration.h>
```

Comment:
I am not sure here if to wrap it with more conditions like the ones below - I need to check if it exists on targets like iOS. If not, the extra `#if` wrapping will be needed (or another, shared `#define` created elsewhere).